server/src/main/java/org/eclipse/lsp/cobol/core/visitor/VariableUsageDelegate.java

```diff
@@ -99,7 +110,7 @@ public void handleGeneralCall(String dataName, Locality locality, List<String> p
         variable.addUsage(variableUsage.locality);
         variablesByUsages.put(variableUsage.locality, variable);
         addParentVariablesUsage(variableUsage.parentVariables, variable);
-      } else {
+      } else if (isNotImplictlyDefinedVariable(variableUsage.name)) {
```

Comment:
How is about checking that an appropriate copybook was included?
I mean if you don't have `COPY SQLDA` or something like that, variable usages must give you an error, I guess.
Maybe we need to hardcode copybooks with that names and variables instead?
As a bonus for hardcoded copybooks, we will have proper hover information.

Reply:
That is the plan for 2-part